examples/plot_kernel_ridge_regression.py

```diff
@@ -136,9 +137,9 @@
         test_time.append(time.time() - t0)
 
     plt.plot(sizes, train_time, 'o-', color="r" if name == "SVR" else "g",
-             label="%s (train)" % name)
+             markeredgecolor=(0, 0, 0), label="%s (train)" % name)
```

Comment:
Totally Agree with you, But whatever changes I have made with a reference of stable version: 
<img src="http://scikit-learn.org/stable/_images/sphx_glr_plot_kernel_ridge_regression_002.png"/> 
As I compared it with Dev version I noticed a slightly difference so I thought it's better to add edgecolor though both look ok.